### PR TITLE
Fibonacci number Implementation in JS. Closes #1609

### LINKED ISSUE #1609: Adding Fibonacci Numbers in JS.

### PR DESCRIPTION
**Pull request by JanaSabuj**
Fixes #1609 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Created JS implementation of the said problem.

#### Languages Used:
- JavaScript

#### Files Added:
- Fibonacci_Number.js

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by JanaSabuj** [comment]
@singh-shreya6 Please do review it

**Comment by JanaSabuj** [comment]
@singh-shreya6 @asha15 @SakshiShreya please do review it.

File: Fibonacci_Number/Fibonacci_Number.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+// Call the functions
+let n = 6;
+console.log(`The ${n}th Fibonacci Number is ${FibonacciNumberRec(n)}`);
+n = 8;
+console.log(`The ${n}th Fibonacci Number is ${FibonacciNumberDP(n)}`);
+n = 10;
+console.log(`The Fibonacci Series upto ${n}th term is ${FibonacciSeries(n)}`);
+
+// I/O:
```
Comment (SakshiShreya) [inline Fibonacci_Number/Fibonacci_Number.js]: Write 'Input' as you have written 'Output'. Be consistent.

**Comment by JanaSabuj** [comment]
@SakshiShreya @asha15  please review it.

**Comment by JanaSabuj** [comment]
@SakshiShreya @asha15 @jainaman224 Squashed commits and made your review changes. Please review and merge the PR.

**Pull request merged**